### PR TITLE
Disable opening of 'historic' notice of works

### PR DESCRIPTION
**Pull request by nayr974**

### CODE REVIEW

File: services/core-web/src/tests/components/mine/NoticeOfWork/__snapshots__/MineNoticeOfWorkTable.spec.js.snap [hunk 1]
```diff
@@ -62,6 +62,7 @@ exports[`MineNoticeOfWorkTable renders properly 1`] = `
     Array [
       Object {
         "document": Object {},
+        "is_historic": undefined,
```
Comment (JustinMacaulay) [inline services/core-web/src/tests/components/mine/NoticeOfWork/__snapshots__/MineNoticeOfWorkTable.spec.js.snap:65]: I think you are just missing this value in the mocked data.

**Review by JustinMacaulay — COMMENTED** [review]
Just a comment on the tests and what we talked about on the AQL for the view.

File: services/core-web/src/tests/components/mine/NoticeOfWork/__snapshots__/MineNoticeOfWorkTable.spec.js.snap (hunk 1 above)
Comment (nayr974) [reply services/core-web/src/tests/components/mine/NoticeOfWork/__snapshots__/MineNoticeOfWorkTable.spec.js.snap:65]: updated!

**Review by JustinMacaulay — APPROVED** [review]
Looks good!

**Pull request merged**